docs/configuration.md

```diff
@@ -135,6 +139,11 @@ the appId, the private key or the certificate for authenticating the extension a
 - `datafeed` contains information of the datafeed service to be used by the bot.
 - `retry` contains information for retry mechanism to be used by the bot.
 
+Although not recommended for RSA private keys, you can specify paths to classpath resources for the following fields:
```

Comment:
maybe we should say that we support only absolute classpath resources? (otherwise it is relative to the BDK package and it does not make sense for users)